examples/loitering/main.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import vqpy
+from vqpy.detector.logger import register
+from yolox_detector import YOLOXDetector
+
+
+def make_parser():
+    parser = argparse.ArgumentParser("VQPy Demo!")
+    parser.add_argument("--path", help="path to video")
+    parser.add_argument(
+        "--save_folder",
+        default=None,
+        help="the folder to save the final result",
+    )
+    parser.add_argument(
+        "-d", "--pretrained_model_dir", help="Directory to pretrained models"
+    )
+    return parser
+
+
+class Person(vqpy.VObjBase):
+    pass
+
+
+class People_loitering_query(vqpy.QueryBase):
+    @staticmethod
+    def setting() -> vqpy.VObjConstraint:
+        REGION = [
+            (550, 550), (1162, 400), (1720, 720), (1430, 1072), (600, 1073)
+            ]
+        REGIONS = [REGION]
+
+        filter_cons = {
+            "__class__": lambda x: x == Person,
+            "bottom_center": vqpy.utils.continuing(
+                condition=vqpy.utils.within_regions(REGIONS),
+                duration=10, name="in_roi"
+            ),
+        }
+        select_cons = {
+            "track_id": None,
+            "coordinate": lambda x: str(x),  # convert to string for
+                                             # JSON serialization
+            # name in vqpy.continuing + '_periods' stored in VObj
+            # can be accessed by getv, be used in select_cons, etc.
```

Comment:
Shall we not mention "be accessed by getv", since `getv` is an internal implementation logic?